mxcube3/ui/reducers/queue.js

```diff
@@ -145,17 +145,18 @@ export default (state = initialState, action) => {
 
     // Adding sample to queue
     case 'ADD_SAMPLE': {
-      const sampleList = { ...state.sampleList, [action.sampleID]: action.sampleData || {} };
+      const sampleList = { ...state.sampleList };
 
       return Object.assign({}, state,
         {
-          todo: { ...state.todo, nodes: state.todo.nodes.concat(action.queueID) },
-          queue: { ...state.queue, [action.queueID]: [] },
-          lookup: { ...state.lookup, [action.queueID]: action.sampleID },
-          lookup_queueID: { ...state.lookup_queueID, [action.sampleID]: action.queueID },
-          collapsedSample: { ...state.collapsedSample, [action.queueID]: true },
-          manualMount: { ...state.manualMount, id: state.manualMount.id + 1 },
-          sampleList
+          displayData: { ...state.displayData, [action.sampleID]: { collpased: false,
```

Comment:
Change to collapsed. There seems to be some problems creating a new sample when one is already in place, I get a error message containing collapsed but only changing this did not seems to solve it.